app/services/date_parser.rb

```diff
@@ -0,0 +1,13 @@
+module DateParser
+  AMERICAN_REGEX = %r{(?<month>\d{1,2})/(?<day>\d{1,2})/(?<year>\d{4})}.freeze
+
+  # Date parsing with a fallback for american-style Month/Day/Year
+  # since we have legacy data in PII bundles that may be stored this way
+  def self.parse(str, allow_american: true)
+    if allow_american && (m = str.match(AMERICAN_REGEX))
+      Date.parse("#{m[:year]}-#{m[:month]}-#{m[:day]}")
```

Comment:
Wasn't sure if this would handle e.g. `Date.parse('2020-1-1')` which could technically match from the regex. Seems fine though in quick testing 👍  At most worth a test case, but I think it's fine as-is.

Reply:
added test case in d8f256a00f73127d728b2bd810a8400c58ec7eed